src/cli/cmd/farm/update/cmd.go

```diff
@@ -71,8 +76,27 @@ func cmdRunE(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func setupArtifactDir(repo, existingFilePath string) (string, string, error) {
```

Comment:
These changes already present on main branch. Seems the branch needs to be rebased with main branch.

Reply:
rebasing with main fixed this.